README.md

```diff
@@ -8,10 +8,10 @@ The diagram below shows an example of capturing and processing direct broadcast
 
 ## Deployment Steps
 1. [Register and authorize a spacecraft](https://docs.microsoft.com/en-us/azure/orbital/register-spacecraft) - Creates a spacecraft resource containing the required information to identify the spacecraft and verifies that you are authorized to communicate with it. 
-2. Deploy [tcp-to-blob](/tcp-to-blob/README.md) - Deploys a TCP endpoint for receiving data from the ground station. 
-3. Deploy [aqua-processor](/examples/aqua-processor/README.md) - Creates an Azure VM for processing the downlinked satellite data.
+2. Deploy [tcp-to-blob](/tcp-to-blob/README.md) - Deploys a TCP endpoint for receiving data from the ground station. It is recommended to deploy TCP to BLOB in the same resource group the spacecraft has been deployed and in the same region. If there is a quota issue in that region then choose a different region but still deploy it in the same resource group. 
```

Comment:
Would it be worth mentioning why it is recommended?

Reply:
After having a discussion with AOGS team, I think it is not required to have both Spacecraft & TCP to Blob under the same RG. They've been running each in a separate RG and able to schedule and process contact data. I'm going to test this approach and amend the PR accordingly.

Reply:
I deployed TCP to BLOB under a new RG, Spacecraft is able to pick up the CP from the deployment and the contact data has been successfully processed. I have updated the PR accordingly.